tests/VirtoCommerce.Platform.Tests/Modularity/ModuleInstallerUnitTests.cs

```diff
@@ -104,7 +104,7 @@ public IEnumerator<object[]> GetEnumerator()
                     new[] { new ModuleManifest { Id = "A", Version = "3.5.0", PlatformVersion = "3.0.0", Dependencies = new []{ new ManifestDependency { Id = "B", Version = "3.5.0" } }} },
                     new[]
                     {
-                        //installed
+                        //installed. Don't check dependency version
```

Comment:
could you mock up IExternalModulesClient.CompleteListWithDependencies with dependencies special for this cases?